client/js/stores/snippet-store.js

```diff
@@ -11,6 +11,7 @@ class SnippetStore {
     this.bindActions(RatingActions);
     this.state = {
       snippets: [],
+      currentUserSnippets: [],
```

Comment:
We've talked before about moving user snippets to separate store, are you planning to do this?